### ISSUE TITLE
Some documents generates pdf some txt, pdf wanted

### ISSUE THREAD

**Issue by rabdavinci** [ritiek/scribd-downloader#23]
1) This generates pdf scribdl  https://ru.scribd.com/document/371974195/Nicholas-Carr-A-Geracao-Superficial-O-Que-a-Internet-Esta-Fazendo-Com-Nossos-Cerebros-1

2) This generates txt scribdl  https://ru.scribd.com/document/175330313/How-to-Develop-and-Write-a-Grant-Proposal

I wanted get the second one pdf format too. Any idea?

Thanks a lot

**Comment by ritiek** [comment]
I made some changes to the PDF thing in #25. Update your installation with:
```
$ pip install git+https://github.com/ritiek/scribd-downloader.git -U
```

and then pass the `-p` option explicitly for generating PDF, like so:
```
$ scribdl -p https://ru.scribd.com/document/175330313/How-to-Develop-and-Write-a-Grant-Proposal
```

No PDF will be generated if `-p` isn't passed.

I'll release a new version once #26 happens.

**Comment by rabdavinci** [comment]
Ritiek, good work! Fixed!
Thank you a lot 👍 

Cordially!

**Comment by ritiek** [comment]
Good to know! Going to close this as I just released [v1.0.0](https://github.com/ritiek/scribd-downloader/releases/tag/v1.0.0).
